spyder/config/main.py

```diff
@@ -456,6 +456,12 @@
               'variable_explorer/copy': 'Ctrl+C',
               # ---- In widgets/plots/figurebrowser.py ----
               'plots/copy': 'Ctrl+C',
+              # ---- In widgets/explorer ----
+              'explorer/copy file': 'Ctrl+C',
+              'explorer/paste file': 'Ctrl+V',
+              'explorer/copy absolute path': 'Ctrl+Alt+C',
+              'explorer/copy relative path 1': 'Ctrl+K',
+              'explorer/copy relative path 2': 'Ctrl+Shift+C',

```

Comment:
@Khalilsqu, why did you add two shortcuts for the same action?

Reply:
![image](https://user-images.githubusercontent.com/22060413/51061597-b7ffc000-15eb-11e9-8b84-8b79196487f4.png)

Was trying to make it similar to vscode:!!!. Shall revert back to what was before?!

Reply:
Yes, please. Also, change the shortcut for relative paths to `Ctrl+Shift+Alt+C` (the same one as in VSCode). The thing is we reserve `Ctrl+Shift+<..>` to switch among plugins.

Reply:
Okay.. `Ctrl+Shift+Alt+C` is the one also used in Pycharm for relative path.